framework/packages/abstract-std/src/native/ibc_client.rs

```diff
@@ -121,6 +121,9 @@ pub enum ExecuteMsg {
         msg: Binary,
         /// Callback info to identify the callback that is sent (acts similar to the reply ID)
         callback_info: Option<CallbackInfo>,
+        /// Set this field in case message is query, defaults to `false`
+        /// Don't forget to set callback_info to catch response
+        is_query: Option<bool>,
```

Comment:
@CyberHoward It's an ugly message, but we have to be careful with new ser/de types. IMO if we slap some helpers for `IbcClient` that would be enough to make it nice